modules/@apostrophecms/schema/ui/apos/components/AposArrayEditor.vue

```diff
@@ -388,28 +388,22 @@ export default {
   }
 
   .apos-modal-array-items {
-    margin: 10px;
+    margin: $spacing-base;
   }
 
   .apos-modal-array-items__heading {
     position: relative;
-    margin: 20px 4px;
-  }
-
-  // Specificity needed due to AposButton rules
-  .apos-modal-array-items__add.apos-button {
```

Comment:
There was no reason for this to be position absolute.  Fixed while applying type